apps/common/management/commands/check_api.py

```diff
@@ -73,6 +73,7 @@ def get_api_urls():
     "/api/v1/authentication/password/reset-code/",
     "/api/v1/authentication/login-confirm-ticket/status/",
     "/api/v1/authentication/mfa/select/",
+    "/api/v1/authentication/mfa/face/context/",
     "/api/v1/authentication/mfa/send-code/",
     "/api/v1/authentication/sso/login/",
     "/api/v1/authentication/user-session/",
```

Comment:
There are several differences between this version (v1) and another version of the code:

  * The `/api/v1/authentication` prefix is removed from some endpoints (e.g., `/api/v1/auth/token/get/`, `/api/v1/authentication/login-confirm-ticket/`). This might be due to changes in server architecture or requirements for authentication.

  * There's a difference in endpoint naming: "/authentication/password/reset-code/" is removed while "login" becomes both prefixes ("sso") in other parts which may affect compatibility with existing systems that rely on specific structure.

The main issue I see is regarding `auth.api.v1()` leading into various sub-endpoints without using "/" in their paths, instead, it seems there could potentially conflict with existing patterns used elsewhere if not managed correctly. It would help maintain backwards-compatibility to ensure these URLs have proper slash after authentication if it were necessary to differentiate them from old endpoints that use "/" as per previous versions.